### PR TITLE
Update the Node Count for data-path tests on ROSA

### PR DESCRIPTION
**Pull request by krishvoor**

### CODE REVIEW

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml [hunk 1]
```diff
@@ -73,9 +59,9 @@ tests:
       READY_WAIT_TIMEOUT: 30m
       REPLICAS: "24"
     test:
-    - chain: openshift-qe-node-density
+    - ref: openshift-qe-cluster-density-v2
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml:62]: to keep consistency do we want to use the `chain: openshift-qe-ocp-qe-perfscale-ci-tests` here instead?

Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml:62]: Let us do it as part of https://github.com/openshift/release/pull/42602 for consistency, unless we have a dependency here.

Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml]: Will keep this PR to update the node-count, will tackle the ROSA changes seperately.

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-perfscale-rosa-multiaz-data-path-9nodes-periodic

**Pull request merged**